apps/wallet-ui/.gitignore

```diff
@@ -26,7 +26,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env*
+.env*.local
```

Comment:
instead exclude `.env.example` so that people like me with their `.env` files don't end up checking them in